BestFloww/BestFlow
# Adding BE post override

## Pull Request Summary
Added override field which when false makes the dao check if the project_id already exists. if it does, it rejects the post.

### Change types
- [x] Breaking change (new feature which messes with old one)

### Testing

### How to manually test this PR (QA instructions)
Check MongoDB to see if there are intents with project_id 1. If there aren't. Then post should work as normal. Otherwise, post our test.json (notice all project_ids are 1) and verify that you get the expected error back in the console.

## Questions
Please test this code both manually and review the algorithms, specifically for efficiency.
